<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarginFilterControllerWidget</class>
 <widget class="QWidget" name="MarginFilterControllerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>Border Filter</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QRadioButton" name="cuttingRadioButton">
       <property name="text">
        <string>Cut invalid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="toNaNRadioButton">
       <property name="text">
        <string>Invalid to NaN</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="xPassRadioButton">
       <property name="text">
        <string>X Pass</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="xStopRadioButton">
       <property name="text">
        <string>X Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>X low</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="xLowMarginLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>X high</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="xHighMarginLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="yPassRadioButton">
       <property name="text">
        <string>Y Pass</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="yStopRadioButton">
       <property name="text">
        <string>Y Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Y low</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="yLowMarginLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Y high</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="yHighMarginLineEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
